docs/source/index.md

```diff
@@ -2,49 +2,85 @@
 hide-toc: true
 ---
 
-```{eval-rst}
-.. image:: ../images/banner.svg
-  :width: 600
-  :alt: Alternative text
-```
+![](../images/top_camera.gif)
```

Comment:
Frank suggested that we replace this with a row of three smaller gifs to make it look like a banner.

Reply:
Should we use existing gifs (the ones in the slides for example) or generate new ones?

Reply:
Only do this if you have easy access to such gifs already. If they need to be generated, we'll worry about that later.